### PR TITLE
Fixes for osg-topology and topology_utils (SOFTWARE-3446)

### PR DESCRIPTION
**Pull request by matyasselmeci**
- Add the `src` directory to `sys.path` so `bin/osg-topology` finds `topology_utils`
- Never cache the results so we don't get too little info when following up an unauth query with an auth query (or worse, too much info when following up an auth query with an unauth query)

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Yes to both.

**Comment by matyasselmeci** [comment]
OK, _now_ it works.

File: src/topology_utils.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def get_auth_session(args):
         key = args.key
 
     session = requests.Session()
+    session.headers.update({'Cache-Control': 'no-cache, no-store, must-revalidate'})
```
Comment (brianhlin) [inline src/topology_utils.py]: Do we still need this with the `no_proxy` env var?

**Comment by brianhlin** [comment]
After you merge this, can you update the following docs?

- https://opensciencegrid.org//operations/services/sending-announcements/
- https://opensciencegrid.org/technology/release/cut-sw-release/
- https://opensciencegrid.org/technology/release/cut-data-release/

**Pull request merged**